src/instructlab/configuration.py

```diff
@@ -172,14 +188,21 @@ class _serve(BaseModel):
     model_path: StrictStr
 
     # additional fields with defaults
-    host_port: StrictStr = "127.0.0.1:8000"
+    host: str
+
+    port: str
+
+    # host_port: StrictStr = "127.0.0.1:8000"
     backend: Optional[str] = (
         None  # we don't set a default value here since it's auto-detected
     )
 
+    additional_args: dict[str, str]
```

Comment:
Are we always guaranteed to be mapping to a `str`? If not, I would change this to `dict[str, typing.Any]`, else you can ignore this comment.

Reply:
oh honestly good point I was thinking of doing this